src/traits.c

```diff
@@ -385,7 +385,7 @@ Expression *pointerBitmap(TraitsExp *e)
             d_uns64 arrayoff = offset;
             d_uns64 nextsize = t->next->size();
             d_uns64 dim = t->dim->toInteger();
-            for (size_t i = 0; i < dim; i++)
+            for (size_t i = 0; i < (size_t)dim; i++)
```

Comment:
Actually I went the other way.  The `dinteger_t`/`size_t` code pretty much assumes length doesn't exceed `2^^32`.